app/lib/search/query_builder.rb

```diff
@@ -255,185 +440,8 @@ def licence_transaction_path?
       finder_content_item.base_path == "/find-licences"
     end
 
-    def stopwords
-      generic_stopwords = %w[
-        a
-        about
-        above
-        after
-        again
-        against
-        all
-        am
-        an
-        and
-        any
-        are
-        arent
-        as
-        at
-        be
-        because
-        been
-        before
-        being
-        below
-        between
-        both
-        but
-        by
-        cant
-        cannot
-        could
-        couldnt
-        did
-        didnt
-        do
-        does
-        doesnt
-        doing
-        dont
-        down
-        during
-        each
-        few
-        for
-        from
-        further
-        had
-        hadnt
-        has
-        hasnt
-        have
-        havent
-        having
-        he
-        hed
-        hell
-        hes
-        her
-        here
-        heres
-        hers
-        herself
-        him
-        himself
-        his
-        how
-        hows
-        i
-        id
-        ill
-        im
-        ive
-        if
-        in
-        into
-        is
-        isnt
-        it
-        its
-        itself
-        lets
-        me
-        more
-        most
-        mustnt
-        my
-        myself
-        no
-        nor
-        not
-        of
-        off
-        on
-        once
-        only
-        or
-        other
-        ought
-        our
-        ours
-        ourselves
-        out
-        over
-        own
-        same
-        shant
-        she
-        shed
-        shell
-        shes
-        should
-        shouldnt
-        so
-        some
-        such
-        than
-        that
-        thats
-        the
-        their
-        theirs
-        them
-        themselves
-        then
-        there
-        theres
-        these
-        they
-        theyd
-        theyll
-        theyre
-        theyve
-        this
-        those
-        through
-        to
-        too
-        under
-        until
-        up
-        very
-        was
-        wasnt
-        we
-        wed
-        well
-        were
-        werent
-        weve
-        what
-        whats
-        when
-        whens
-        where
-        wheres
-        which
-        while
-        who
-        whos
-        whom
-        why
-        whys
-        with
-        wont
-        would
-        wouldnt
-        you
-        youd
-        youll
-        youre
-        youve
-        your
-        yours
-        yourself
-        yourselves
-      ]
-
-      brexit_stopwords = %w[abroad brexit eu europe european exit leave union]
-
-      (generic_stopwords + brexit_stopwords).uniq
+    def brexit_path?
+      finder_content_item.base_path == "/find-eu-exit-guidance-business"
```

Comment:
If the brexit stopwords were added just for one finder, do we even need them anymore? `/find-eu-exit-guidance-business` now redirects to a collection page https://www.gov.uk/government/collections/brexit-guidance.

Reply:
Have removed brexit stopwords. I've also cut out the generic stopwords - I think your second plan is the best, let's remove them for now and see what happens with the new search team. We can add back the generic stopword list if it turns out it's useful.